src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -42,10 +42,10 @@ object ExtractionPipeline {
     /**
       * Generic helper method for extracting linked entities and saving them.
       *
-      * @param encodeEntity the entity from which to extract a field
-      * @param referenceField the field name to use as an identifier in getIds
-      * @param prevData the data to base the ID extraction off of
-      * @param entityFieldName the field name to extract if it isn't @id
+      * @param encodeEntity the entity which should be extracted
+      * @param referenceField the field name to use as an identifier for the encodeEntity (in getIds)
+      * @param prevData the parent entity (which references a set of encodeEntities) from which ID extraction will be based
+      * @param entityFieldName the encodeEntity field name which references the parent (default is @id)
```

Comment:
Existing code, but: I think these argument names could be much more clear. Can we rename them? I'm thinking something like:
* `encodeEntity` -> `entityToExtract`
* `referenceField` -> `matchingField`
* `prevData` -> `linkingEntities`
* `entityFieldName` -> `linkField`

Reply:
(Also, to be clear: naming debates are open to all. Please counter-suggest if you don't like one of mine 😄)

Reply:
I like your suggestions 😁... I had been struggling to come up with something to replace prevData

Reply:
Although I might change `linkField` to `linkedField` unless you object